catalog/solr/pom.xml

```diff
@@ -30,6 +30,11 @@
                 <artifactId>joda-time</artifactId>
                 <version>2.7</version>
             </dependency>
+            <dependency>
+                <groupId>commons-fileupload</groupId>
+                <artifactId>commons-fileupload</artifactId>
+                <version>1.3.2</version>
```

Comment:
Might be safer with respect to solr's transitive dependencies. If it works as-is, when we upgrade that doesn't mean we want to upgrade solr too. Might introduce a bug.